spec/javascripts/health-care/utils/validations.spec.js

```diff
@@ -59,4 +59,17 @@ describe('Validations unit tests', () => {
       expect(isValidDate(0, 2, 2016)).to.be.false;
     });
   });
+
+  describe('isValidMonetaryValue', () => {
+    it('validates monetary values', () => {
+      expect(isValidMonetaryValue('100')).to.be.true;
+      expect(isValidMonetaryValue('1.99')).to.be.true;
+      expect(isValidMonetaryValue('1000')).to.be.true;
+      expect(isValidMonetaryValue('')).to.be.true;
```

Comment:
I disagree with that. That's tying the validation API to the use case too much.  Someone could compose that with `isBlank() || isValidMonetaryValue()` nearly as easily.